cli/azd/cmd/package.go

```diff
@@ -47,9 +47,11 @@ func (pf *packageFlags) Bind(local *pflag.FlagSet, global *internal.GlobalComman
 
 func newPackageCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:    "package <service>",
-		Short:  "Packages the application's code to be deployed to Azure.",
-		Hidden: true,
+		Use: "package <service>",
+		Short: fmt.Sprintf(
+			"Packages the application's code to be deployed to Azure. %s",
+			output.WithWarningFormat("(BETA)"),
```

Comment:
is `BETA` a thing? @savannahostrowski  ?   IIRC, we have alpha -> preview -> GA

Reply:
She is the one that recommended (BETA).  Will follow-up offline.

Reply:
@savannahostrowski confirmed it should be `(Beta)` here.